diff --git a/themes/gallois.zsh-theme b/themes/gallois.zsh-theme
index 515325e3..537a5c44 100644
--- a/themes/gallois.zsh-theme
+++ b/themes/gallois.zsh-theme
@@ -20,4 +20,5 @@ ZSH_THEME_RVM_PROMPT_SUFFIX="]%{$reset_color%}"
 # Combine it all into a final right-side prompt
 RPS1='$(git_custom_status)$(ruby_prompt_info) $EPS1'
 
-PROMPT='%{$fg[cyan]%}[%~% ]%(?.%{$fg[green]%}.%{$fg[red]%})%B$%b '
+#PROMPT='%{$fg[cyan]%}[%~% ]%(?.%{$fg[green]%}.%{$fg[red]%})%B$%b '
+PROMPT='%{$fg[white]%}[%~% ]%(?.%{$fg[green]%}.%{$fg[red]%})$ %{$reset_color%}'
